src/applications/disability-benefits/all-claims/config/schema.js

```diff
@@ -786,6 +786,9 @@ const schema = {
       type: 'string',
       enum: serviceBranches,
     },
+    hasTrainingPay: {
```

Comment:
Ahh, okay. So this is not intended to be sent to the back end? :+1: 

Should this be added to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15004?

Reply:
Yes, and yes :)

It doesn't strictly _need_ to be sent up (since it can be deduced), but it's a lot simpler to just send the value up than to have the backend deduce it with code.

Reply:
[Added to that ticket]